package/MDAnalysis/core/topology.py

```diff
@@ -151,8 +151,12 @@ class TransTable(object):
 
     Parameters
     ----------
-    n_atoms, n_residues, n_segments : int
```

Comment:
I think it still works in some places. We used it more often. But there also isn't a good way to look at the intermediate rEst to know what exactly is going on.